.github/workflows/docs.yml

```diff
@@ -16,13 +17,29 @@ jobs:
     permissions:
       contents: write
       pull-requests: write
+    env:
+      BIN_DIR: ${{ github.workspace }}/bin
+      DOXYGEN_URL: 'https://github.com/doxygen/doxygen/releases/download/Release_1_10_0/doxygen-1.10.0.linux.bin.tar.gz'
+      DOXYGEN_VERSION: 1.10.0
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup
 
       - name: Install dependencies
         run: |
-          sudo apt-get install -y doxygen python3-msgpack
+          sudo apt-get install -y python3-msgpack
```

Comment:
could we perhaps fail CI if apt gets >= 1.9.5, and print a message that reminds us to switch back to apt instead of building from the tarball?